.travis.yml

```diff
@@ -27,6 +27,6 @@ jobs:
         npm run sass-lint -- --max-warnings 0 &&
         npm run test &&
         npm run build
-      script: npm run build:ci &&
+    - script: npm run build:ci &&
```

Comment:
👀 

So, what's happening with this then? Did I accidentally revert this?

Reply:
Not a big fan of pinning the blame on people - but in this case I think it was me!